Engine/source/gfx/gl/tGL/tGL.h

```diff
@@ -24,7 +24,8 @@
 #define T_GL_H
 #include "GL/glew.h"
 
-#define gglHasExtension(EXTENSION) GLEW_##EXTENSION
+// Slower but reliably detects extensions
+#define gglHasExtension(EXTENSION) glewGetExtension("GL_" # EXTENSION)
```

Comment:
Not exactly a solution I'm in love with, but 
#if defined (TORQUE_OS_WIN)
#define gglHasExtension(EXTENSION) GLEW_##EXTENSION
#else
// Slower but reliably detects extensions
#define gglHasExtension(EXTENSION) glewGetExtension("GL_" # EXTENSION)
#endif

seems to work :/